ezpublish/googlenewsbundle/composer.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "transfer/GoogleNewsBundle",
+  "type": "library",
+  "description": "Example bundle for transfer/ezplatform",
+  "keywords": ["transfer", "extension", "manifest", "worker"],
+  "license": "MIT",
+  "minimun-stability": "dev",
+  "authors": [
+    {
+      "name": "Harald Tollefsen",
+      "email": "harald@tollefsen.info"
+    }
+  ],
+  "require": {
+    "php": "~5.5|~7.0",
+    "symfony/yaml": "~2.7",
+    "monolog/monolog": "~1.17",
+    "doctrine/orm": "^2.5",
+    "transfer/ezplatform": "1.0.x-dev"
```

Comment:
I could/should have pointed this toward the beta 2maybe. On the the other hand I think the examples should be working towards the latest dev-release until a stable release of transfer/ezplatform is out